db.go

```diff
@@ -862,6 +868,7 @@ func (db *DB) sendToWriteCh(entries []*Entry) (*request, error) {
 		size += e.estimateSizeAndSetThreshold(db.valueThreshold())
 		count++
 	}
+	y.NumBytesWrittenUserAdd(db.opt.MetricsEnabled, int64(size))
```

Comment:
I notice this is the estimate of bytes that will be sent to a channel. How different is this from bytes written to disk? Should we clarify the metric name if this channel may back up, or if it is substantially different than what is written to disk?